### PR TITLE
Added Travis badge

### PR DESCRIPTION
**Pull request by aarushgupta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/aarushgupta/autowebcompat.svg?branch=travisbadge)](https://travis-ci.org/aarushgupta/autowebcompat)
```
Comment (poush) [inline README.md]: That's the wrong url. It should of this repository.

Comment (poush) [reply README.md]: and branch should be master.

Comment (aarushgupta) [reply README.md]: Could you please tell me the travis url of this repository?

Comment (aarushgupta) [reply README.md]: Will change the required things

**Pull request closed without merging**